README.md

```diff
@@ -69,10 +70,11 @@ http://www.ubuntu.com/legal/contributors
 To get the source and propose a merge, this is what typically needs to
 be done:
 
-     bzr branch lp:snappy my-work
-     cd my-work
-     [hack on mywork]
-     bzr lp-propose
+    cd ~/work/src/github.com/ubuntu-core/snappy
+    git checkout -b my-work
+    [hack on mywork]
+    git push
+    [create pull request]
```

Comment:
patches welcome! The bzr thing is clearly wrong and needs fixing. I don't understand how to translate what you wrote into the format used by the README, so go ahead.